pkg/pub_package_reader/lib/pub_package_reader.dart

```diff
@@ -418,3 +431,19 @@ Iterable<ArchiveIssue> requireIosFolderOrFlutter2_20(
         '^1.20.0 or higher (environment.sdk.flutter) or create an `ios/` folder.\n\nSee $_pluginDocsUrl');
   }
 }
+
+Iterable<ArchiveIssue> requireNonEmptyLicense(
+    String path, String content) sync* {
+  if (path == null) {
+    yield ArchiveIssue('LICENSE file not found.');
+    return;
+  }
+  if (content == null || content.trim().isEmpty) {
+    yield ArchiveIssue('LICENSE file `$path` has no content.');
+    return;
+  }
+  if (content.contains('TODO: Add your license here.')) {
```

Comment:
Can we make this: `content.trim().toLowerCase() == "TODO: Add your license here.".toLowerCase())`